src/styles/About.css

```diff
@@ -88,8 +81,11 @@
 }
 
 .RightBlurbImages {
+  float: left;
   width: 80px;
-  position:absolute;
+  border-radius: 15px;
+  background-color: white;
+  position: absolute;
```

Comment:
About.css: Combination of two `.RightBlurbImages` improves organization of code for future additions/amendments. There were two `width: 80px;`, but one of them has been removed.